spring-gemfire-extensions/src/main/java/org/springframework/geode/distributed/event/MembershipListenerAdapter.java

```diff
@@ -54,9 +51,6 @@ public final void memberDeparted(DistributionManager manager, InternalDistribute
 
 	public void handleMemberDeparted(MemberDepartedEvent event) { }
 
-	/**
-	 * @inheritDoc
-	 */
```

Comment:
What is the effect of removing this javadoc annotation?

Reply:
I'm not sure, javadoc wasn't able to build without removing it. It just complained that it was an invalid use.

Reply:
Please investigate the before and after

Reply:
I've got building with the inheritDoc annotations in place. Removing them removed some important information, but now the docs are identical to before.